src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,13 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
-        return null;
+    public static User findByEmail(String email) {
+        User result = null;
+        for (User user : users) {
+            if (user.getEmail().equals(email)) {
+                result = user;
+            }
+        }
+        return result;
```

Comment:
You can optimize it like this 
```suggestion
        for (User user : users) {
            if (user.getEmail().equals(email)) {
                return user;
            }
        }
        return null;
```

Reply:
Hello!
It was so as first everywhere, but IntellijIdea have proposed such fixes.
Working on this now.
Thanks!